README.rst

```diff
@@ -115,7 +116,7 @@ the Unit Test section).
 
 Contact
 -------
-f5_common_python@f5.com
+Join us on `Slack <https://f5-openstack-slack.herokuapp.com/>`_!
```

Comment:
Hi Jode, lets do this also via the badge like the docs at the top of the page.  There are some examples here: https://github.com/F5Networks/slackin